SCClassLibrary/Common/Control/Server.sc

```diff
@@ -141,7 +141,7 @@ ServerOptions {
 			o = o ++ " -U " ++ if(ugenPluginsPath.isString) {
 				ugenPluginsPath
 			} {
-				ugenPluginsPath.join("; ");
+				ugenPluginsPath.join(Platform.pathDelimiter);
```

Comment:
User-specified paths might include spaces or other special characters. These should be escaped, or the paths quoted. IIRC Windows requires quoting.

```
-U my\ custom\ path:other\ path

or

-U 'my custom path':'other path'

or this? I don't know if the above or below is correct

-U 'my custom path:other path'
```

For that matter, I don't know if the paths should be quoted individually, or if the entire -U string should be quoted outside (with `:` or `;` inside the quotes).

Reply:
this is addressed